Project.toml

```
@@ -1,7 +1,7 @@
 name = "TrixiAtmo"
 uuid = "c9ed1054-d170-44a9-8ee2-d5566f5d1389"
-authors = ["Benedict Geihe <bgeihe@uni-koeln.de>"]
-version = "0.1.0"
+authors = ["Benedict Geihe <bgeihe@uni-koeln.de>", "Tristan Montoya <montoya.tristan@gmail.com>"]
+version = "0.1.1-dev"
```

```suggestion
version = "0.1.0-DEV"
```

We should reserve 0.1.0 for the first release

---

So should this then be "0.0.1-DEV" or something like that?

---

`0.1.0-DEV` is fine!
Once we do our first release it will be be `0.1.0` and we will then work on `0.1.1-DEV`.

---

Ok, that makes sense! Thanks Hendrik and Benedict!